boot/seal.go

```diff
@@ -147,14 +148,25 @@ func runKeySealRequests(key keys.EncryptionKey) []secboot.SealKeyRequest {
 	}
 }
 
+// FallbackSaveSealedKeyUnder returns the name of a fallback ubuntu save key.
+func FallbackSaveSealedKeyUnder(dir string) string {
+	return filepath.Join(dir, "ubuntu-save.recovery.sealed-key")
+}
+
+// FactoryResetFallbackSaveSealedKeyUnder returns the name of a fallback ubuntu
+// save key object generated during factory reset.
+func FactoryResetFallbackSaveSealedKeyUnder(dir string) string {
```

Comment:
are we going to call this from other packages?

Reply:
yes, it's called in devicestate in #11870